.github/deploy.yml

```diff
@@ -55,7 +55,8 @@ conda:
         license: MIT
         license_family: MIT
         license_file: LICENSE
-        summary: ORM interface for the Clingo ASP Solver
+        summary: A visualizer for graphs defined as a set of facts with special integration for clingo
```

Comment:
@rkaminsk Can I change this manually also in `.github/conda/meta.yml` or is that one of the generated files? When are these files generated?

Reply:
No, this script is generated using <https://github.com/potassco/deploy> (a private repository).

Reply:
Ok, I see the right file to change this was the `conda/meta` how do I run this private script to update files like this `deploy.yml`?

Reply:
You just check out the repository. And run `python <path-to-repository>/deploy.py` in clingraph's .github folder. You will rarely have to do this. It's mostly me who updates all the projects if some CI script breaks.